packages/bpk-stylesheets/webpack.config.babel.js

```diff
@@ -123,5 +123,8 @@ module.exports = {
     new MiniCssExtractPlugin({
       filename: 'base.css',
     }),
+    new MiniCssExtractPlugin({
+      filename: 'index.css',
+    }),
```

Comment:
This complies the code to strip out non browser understandable sass such as `global()`